dvc/repo/experiments/executor/local.py

```diff
@@ -143,6 +145,18 @@ def _update_config(self, update):
             with open(local_config, "wb") as fobj:
                 conf_obj.write(fobj)
 
+    def _update_git_config(self, update):
+        local_config = os.path.join(self.scm.root_dir, ".git", "config")
+        logger.debug("Writing experiments local Git config '%s'", local_config)
+        if os.path.exists(local_config):
+            conf_obj = ConfigObj(local_config, list_values=False)
+            conf_obj.merge(update)
+        else:
+            conf_obj = ConfigObj(update, list_values=False)
+        if conf_obj:
+            with open(local_config, "wb") as fobj:
+                conf_obj.write(fobj)
```

Comment:
We could probably simplify this to just copy file. But this may be useful in the future. 
```
try:
	shutil.copyfile(local_git_config, os.path.join(self.scm.root_dir, ".git", "config"))
except FileNotFoundError:
	pass
```

Reply:
That's what I originally had, but I decided to copy what we do for the dvc config (see the lines directly above), although I agree it's probably overkill. Not sure if there's any case when we actually need to keep the default git config that gets generated (there is a default config file generated, so this does hit the merge condition).

Reply:
No strong opinion if you want to change it

Reply:
That's okay for now. I'm not sure if `configobj` can parse all git configs, but we can fix if it becomes a problem.